src/main/scala/xiangshan/cache/mmu/PageTableWalker.scala

```diff
@@ -838,6 +1068,13 @@ class HPTWIO()(implicit p: Parameters) extends MMUIOBaseBundle with HasPtwConst
     val l2Hit = Bool()
     val l1Hit = Bool()
     val bypassed = Bool() // if bypass, don't refill
+    val jmp_bitmap_check = Bool()  //l0 hit in ptwcache, but need bitmap check
+    val pte = UInt(XLEN.W)
+    val ptes = Vec(tlbcontiguous,UInt(XLEN.W))
+    val cfs = Vec(tlbcontiguous,Bool())
+    val hitway = UInt(l2tlbParams.l0nWays.W)
+    val fromSP = Bool()
+    val SPlevel = UInt(log2Up(Level).W)
```

Comment:
Similar to the comments above:

1. Necessary annotations
2. space after comma
3. could you wrap these signals in a Bitmapbundle, with Some(new Bitmapbundle ) when the bitmap is opened, and None otherwise. this might be a bit clearer?